wiki-jpa-dao/src/main/resources/db/changelog/wiki.db.changelog-1.0.0.xml

```diff
@@ -51,11 +52,11 @@
         <constraints nullable="false"/>
       </column>
       <column name="OWNER" type="NVARCHAR(200)"/>
-      <column name="CREATED_DATE" type="TIMESTAMP" defaultValueDate="${now}">
-        <constraints nullable="false"/>
+      <column name="CREATED_DATE" type="TIMESTAMP">
```

Comment:
MySQL 5.5 allows one column with default timestamp, so we could at least keep it for CREATED_DATE. WDYT ?